tests/python_tests/helpers/format_arg_mapping.py

```diff
@@ -180,6 +180,13 @@ class Mailbox(Enum):
 }
 
 
+class DstSync(Enum):
```

Comment:
```suggestion
class DestSynchronization(Enum):
```
No need to be shy with names. It is easier to read when you're new and looking at the code for the first time.